compiler/next/include/chpl/Frontend/FrontendQueries.h

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2021 Hewlett Packard Enterprise Development LP
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef CHPL_FRONTEND_FRONTENDQUERIES_H
+#define CHPL_FRONTEND_FRONTENDQUERIES_H
+
+#include "chpl/AST/BaseAST.h"
+#include "chpl/AST/Builder.h"
+#include "chpl/AST/Expr.h"
+#include "chpl/AST/ID.h"
+#include "chpl/AST/Location.h"
+#include "chpl/AST/ModuleDecl.h"
+#include "chpl/Queries/Context.h"
+
+#include <vector>
+
+namespace chpl {
+
+// This could alternatively be a class with static methods
+namespace FrontendQueries {
```

Comment:
should we try to stick to snake_case for namespace names? Or at least that's what `chpl` and `ast` suggests to me.

Reply:
That's a reasonable suggestion. I've renamed it to `namespace frontend` on my branch with many other renamings that will form a subsequent PR.

Reply:
PR #17609 will get to it